prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -126,17 +132,21 @@ public void onChange(Entity oldEntity, Entity newEntity) throws FalconException{
                 backlogMetricStore.deleteEntityInstance(newProcess.getName());
                 entityBacklogs.remove(newProcess);
             }
+            Process process = EntityUtil.getEntity(oldEntity.getEntityType(), oldEntity.getName());
```

Comment:
Once the back log gets cleared it will come down to zero and previously emitted values will tell us what was the backlog before it reached to zero.

The graph will basically be a trend of the backlog at a particular point in time.